includes/shortcodes/class-wc-shortcode-my-account.php

```diff
@@ -379,6 +379,15 @@ public static function reset_password( $user, $new_pass ) {
 		if ( ! apply_filters( 'woocommerce_disable_password_change_notification', false ) ) {
 			wp_password_change_notification( $user );
 		}
+		/**
+		 * Fires after the user's password is reset.
+		 *
+		 * @since 4.9.0-dev
+		 *
+		 * @param WP_User $user     The user.
+		 * @param string  $new_pass New user password.
+		 */
+		do_action( 'woocommerce_after_password_reset', $user, $new_pass );
```

Comment:
Hello @mfs-mindsize,

This isn't doesn't solve that issue, since now we have a new hook instead of implementing what WP does, so it will still require people to hook into this new one too.

Reply:
Note that we already trigger a hook after resetting passwords: https://github.com/woocommerce/woocommerce/blob/trunk/includes/class-wc-form-handler.php#L1054